packages/core/src/utils/requestQueue.ts

```diff
@@ -150,6 +178,14 @@ export const createRequestQueue = ({
       queue = new Array();
       lastRequestTime = 0;
     },
+    kill: () => {
+      // NOTE: Should this go in clear or pause instead?
+      clearTimeout(timeout);
+      on = false;
+      queue = new Array();
+      lastRequestTime = 0;
+      abortController.abort();
```

Comment:
Does this work after it runs once? Basically does the abort controller need to be reset?